JSDemos/Demos/ContextMenu/Templates/jQuery/index.js

```diff
@@ -4,14 +4,18 @@ $(() => {
     width: 200,
     target: '#image',
     itemTemplate(itemData) {
-      const template = $('<div></div>');
+      const template = $('<div class="item-template-container"></div>');
+
       if (itemData.icon) {
-        template.append(`<span class="${itemData.icon}"><span>`);
+        template.append(`<span class="${itemData.icon}"></span>`);
```

Comment:
I suggest to add `dx-icon` class to item icon & `dx-icon-spinright` element. In this case the icon size will match default & compact themes and will allow us not to redefine the size using the following style:
```
.dx-menu-item-content span:not(.dx-menu-item-text) {
  font-size: 20px;
}
```

Reply:
Please, check it